Procfile.dev

```diff
@@ -1,3 +1,4 @@
 web: unset PORT && bin/rdbg -O -n -c -- bin/rails server
-css: yarn build:css --watch
+css: bin/yarn build:css --watch
 js: bin/shakapacker-dev-server
+stacks: docker compose up --abort-on-container-exit --build
```

Comment:
Does this force a docker dependency for developers using `bin/dev`?

Reply:
@jcoyne It does. I'll tweak that.